frontend/src/metabase/admin/settings/components/CloudPanel/MigrationInProgress.tsx

```diff
@@ -0,0 +1,113 @@
+import { c, t } from "ttag";
+
+import { useCancelCloudMigrationMutation } from "metabase/api";
+import { useSetting } from "metabase/common/hooks";
+import ExternalLink from "metabase/core/components/ExternalLink";
+import { useToggle } from "metabase/hooks/use-toggle";
+import { useDispatch } from "metabase/lib/redux";
+import { addUndo } from "metabase/redux/undo";
+import { Flex, Text, List, Button, Box, Modal, Progress } from "metabase/ui";
+
+import { MigrationCard } from "./CloudPanel.styled";
+import type { InProgressCloudMigration, InProgressStates } from "./utils";
+import { getCheckoutUrl } from "./utils";
+
+interface MigrationInProgressProps {
+  migration: InProgressCloudMigration;
+}
+
+const progressMessage: Record<InProgressStates, string> = {
+  init: t`Talking to Metabase Cloud...`,
+  setup: t`Talking to Metabase Cloud...`,
+  dump: t`Taking a snapshot of this instance...`,
+  upload: t`Uploading the snapshot to the cloud...`,
+};
+
+export const MigrationInProgress = ({
+  migration,
+}: MigrationInProgressProps) => {
+  const dispatch = useDispatch();
+
+  const readOnly = useSetting("read-only-mode");
+
+  const [isModalOpen, { turnOn: openModal, turnOff: closeModal }] =
+    useToggle(false);
+
+  const [cancelCloudMigration] = useCancelCloudMigrationMutation();
+
+  const handleCancelMigration = async () => {
+    closeModal();
+    await cancelCloudMigration();
+    dispatch(
+      addUndo({
+        icon: "info_filled",
+        message: t`Migration to Metabase Cloud has been canceled.`,
+        undo: false,
+      }),
+    );
+  };
+
+  const checkoutUrl = getCheckoutUrl(migration);
+
+  return (
+    <>
+      <MigrationCard>
+        <Flex gap="sm" align="center">
+          <Text fw="bold">{t`You have started migration to Metabase Cloud`}</Text>
```

Comment:
The copy here is a bit odd. Maybe "You have begun migrating to Metabase Cloud." But that doesn't feel 100% right, because strictly speaking it's not the _user_ who is migrating.

Reply:
How about `You are now migrating to Metabase Cloud`?